content/post/Android_Root101.en.md

```diff
@@ -39,26 +42,28 @@ Place this file inside the folder created after extraction.
 
 * Now open up the command prompt on Windows with **_adminitrator priviliges_**.
 
-    _(To do this press the Windows key on your keyboard and search 'cmd' right click when command prompt comes up and chose 'Run as Administrator')_
+    _(To do that, press the `Windows key` on your keyboard and search 'cmd', right click when command prompt comes up and chose 'Run as Administrator')_
 
 * Now go the directory where the files you downloaded have been stored.
     if you don't know how to use Command Prompt
 
 > `cd <dirname>` is use to go up a directory
```

Comment:
Come on, we talked about this
The file structure is not linear, there is no going up a directory
Doesn't work that way

```suggestion
> `cd <dirname>` is used to go to the directory named "<dirname>"
```